### DISCUSSION TITLE
TOUGH3 - Parasitic Load and Productivity/Injectivity Indexes

### DISCUSSION THREAD

**Discussion by SHerrboldt** [NatLabRockies/GEOPHIRES-X#322] [Q&A]
Is there a way to see the parasitic load, Productivity Index, or Injectivity Index when modeling with TOUGH3? 
 
The GEOPHIRES output appears to show the input indexes but I would like to see the indexes that TOUGH calculates.

**Comment by softwareengineerprogrammer** [comment]
![image](https://github.com/user-attachments/assets/789646f6-1530-4854-ac6e-0df554f89e95)
